src/category_theory/filtered.lean

```diff
@@ -233,6 +235,21 @@ An arbitrary choice of cocone over `F : J ⥤ C`, for `fin_category J` and `is_f
 noncomputable def cocone (F : J ⥤ C) : cocone F :=
 (cocone_nonempty F).some
 
+variables {D : Type u₁} [category.{v₁} D]
+
+/-- If `C` is filtered, and we have a right adjoint functor `C ⥤ D`, then `D` is filtered. -/
+lemma of_right_adjoint {L : D ⥤ C} {R : C ⥤ D} (h : L ⊣ R) : is_filtered D :=
```

Comment:
Why do you want to make the pair of adjoint functors explicit? Isn't it more flexible to use the typeclass `is_right_adjoint`?

Reply:
I think it's actually more flexible to have the explicit pair of adjoint functors, but I've added the typeclass version too :)